rcp/org.testeditor.rcp4.views.teststepselection/src/org/testeditor/rcp4/views/teststepselector/TestStepSelector.xtend

```diff
@@ -79,7 +92,32 @@ class TestStepSelector {
 
 				})
 		]
+		val commandService = PlatformUI.workbench.getService(ICommandService)
```

Comment:
The injection is a mess in the class but this is not subject of the ticket. See ToDo in line 185. Please create a new ticket.